README.md

```diff
@@ -110,12 +145,16 @@ the processed gene identifiers columns, results from <2.2.2 and >=2.2.2 cannot b
 
 ## Change Log
 
+* 2.2.3
+
+  * Added comparison script `compareSvsTSVs` for `svs*.tsv` files.
+  * Added basic check to shield against yet unfixed potential unnoticable IO-errors in sophia binary (see [Bitbucket PR](https://bitbucket.org/utoprak/sophia/pull-requests/2/safer-file-io/diff))
+  * Refactorings and documentation to improve code clarity
+
 * 2.2.2
 
   * Bugfix in `intersectionCollapsing.py`: Produced columns with `set()` output and lines were put out instead of withheld, resulting in alignment errors in subsequent unchecked paste commands. Bug was fixed, script refactored and Python 3 type hints employed (not enforced by Python, though).
   * Added `safe_paste.py` to shield against similar bugs in the future
-  * Added basic check to shield against yet unfixed potential unnoticable IO-errors in sophia binary (see [Bitbucket PR](https://bitbucket.org/utoprak/sophia/pull-requests/2/safer-file-io/diff))
```

Comment:
is this feature in 2.2.2??? If, i would leave this line here, although it was not intended for this realease.